hack/make.ps1

```diff
@@ -350,22 +359,18 @@ Function Run-IntegrationTests() {
         $xmlFilePath = $bundlesDir + "\junit-report-int-tests-$normDir" + ".xml"
         Set-Location $dir
         Write-Host "Running $($PWD.Path)"
+
+        $goTestArg = "--format=standard-verbose --jsonfile=$jsonFilePath --junitfile=$xmlFilePath -- $env:INTEGRATION_TESTFLAGS"
```

Comment:
Was there a reason for moving this to an intermediate variable?

Reply:
Just syncing style between `Run-UnitTests()` and `Run-IntegrationTests()` functions.

Reply:
Gotcha; yes, there were some of these variables in those as well. Love to get rid of more of those; especially if they're only used once. I had some discussion about that in https://github.com/moby/moby/pull/39998, but in the end didn't want to block / delay merge of it

Reply:
Well if we want start cleanup Windows CI then IMO we should first remove those LCOW and integration inside of container references as those scenarios are not actually used here. I quckly checked it on https://github.com/olljanat/moby/commit/630cbc6b349f569281e3a71b9c3b1f8c2a852fe3 and looks that would let us remove a lot of old logic. Anyway it better do thst on later PR.